src/translator/byteArrayExample.cpp

```diff
@@ -42,4 +42,25 @@ void * getBinaryModelFromConfig(marian::Ptr<marian::Options> options) {
     }
 }
 
+std::vector<char> getBinaryShortlistFromFile(const std::string& filename){
```

Comment:
I want these functions in `marian::bergamot` for testing service. I see this as a marian::bergamot util function. Move the other functions (corresponding to model) as well in the process. Mozilla will directly supply const void* and perhaps size and they have no use for this function in a `bergamot::` namespace. For test-apps like `bergamot-translator-app`, they can pull it from marian anyway.

Reply:
I see your point. Where should I put these function then?

Reply:
Just adjust the namespace to `marian::bergamot`, propogate to where used and name the files in snake case (whatever marian adhere's to). As a general note, anything in `marian::bergamot` has to be named following marian-dev conventions.

Reply:
This isn't even a shortlist function.  It could be named load file to memory.  And it could be a method to some blob class.  

Also, use of std::vector<char> is discouraged because we want to have proper alignment.